Update config_dscp_steering.md (#820)
128technology/docs: docs/config_dscp_steering.md

- Configure a parent service that matches the traffic profile of the tunnel
- Configure a child service that defines the `dscp-range` to steer

:::info

> When configuring DSCP steering it is important to have all possible DSCP values (0-63) configured over the collective of child services that share a parent. Omiting a value, can lead to traffic which matches the parent service but whose DSCP is not covered by a child service failing with a "No ServicePaths available" error.

Reviewer: ```suggestion
When configuring DSCP steering it is important to have all possible DSCP values (0-63) configured over all child services that share a parent. Omitting a value can lead to traffic that matches the parent service, but whose DSCP is not covered by a child service. The result is a service failure with a `No ServicePaths available` error.
```
